### PR TITLE
testCase for prevent unintended parties from modifying/viewing API

### PR DESCRIPTION
**Pull request by HiranyaKavishani**
## Purpose
Add scenario test for preventing unintended parties from modifying/viewing API

### CODE REVIEW

File: product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/java/org/wso2/am/scenario/tests/rest/api/publisherVisibilityRoleRestriction/PublisherAccessControlTestCase.java [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        }
+
+        for (String username : userSet) {
+            if (!username.equals("testUser__1")) {
+                deleteUser(username, TENANT_LOGIN_ADMIN_USERNAME, ADMIN_PASSWORD);
+            }
+            deleteUser(username, ADMIN_LOGIN_USERNAME, ADMIN_PASSWORD);
+        }
+    }
+}
```
Comment (TanyaM) [inline product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/java/org/wso2/am/scenario/tests/rest/api/publisherVisibilityRoleRestriction/PublisherAccessControlTestCase.java]: New line required.

File: product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="ApiManager-scenario-tests-suite">
+    <test name="2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs" preserve-order="true"
+          verbose="2" parallel="false">
+        <classes>
+            <class name="org.wso2.am.scenario.tests.rest.api.publisherVisibilityRoleRestriction.PublisherAccessControlTestCase"/>
+        </classes>
+    </test>
+</suite>
```
Comment (TanyaM) [inline product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/resources/testng.xml]: New line required.

File: product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/pom.xml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+            <name>WSO2 API Manager Developers' List</name>
+            <post>mailto:dev@wso2.org</post>
+            <archive>http://mail.wso2.org/mailarchive/dev/</archive>
+            <subscribe>mailto:dev-request@wso2.org?subject=subscribe</subscribe>
+            <unsubscribe>mailto:dev-request@wso2.org?subject=unsubscribe</unsubscribe>
+        </mailingList>
+    </mailingLists>
+
+
+
```
Comment (TanyaM) [inline product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/pom.xml]: unnecessary new lines?

File: product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/java/org/wso2/am/scenario/tests/rest/api/publisherVisibilityRoleRestriction/PublisherAccessControlTestCase.java (hunk 1 above)
Comment (HiranyaKavishani) [reply product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/2.2.1-prevent-unintended-parties-from-modifying-and-viewing-APIs/src/test/java/org/wso2/am/scenario/tests/rest/api/publisherVisibilityRoleRestriction/PublisherAccessControlTestCase.java]: Fixed with 1ced581ac554b27ba080eadb673460960d8a9381

File: product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/pom.xml (hunk 3 above)
Comment (HiranyaKavishani) [reply product-scenarios/2-collaborative-api-development/2.2-manage-visibility-of-apis-in-publisher-portal/pom.xml]: Fixed with 1ced581ac554b27ba080eadb673460960d8a9381

**Pull request merged**